Resources/Private/Partials/ThumbnailDescription.html

```diff
@@ -24,7 +24,7 @@
     <f:for each="{files}" as="file">
         <tr>
             <td>
-                <f:render partial="ThumbnailDescription/File" arguments="{file: file}" />
+                <f:render partial="ThumbnailDescription/File" arguments="{file: file, newTimestamp, newTimestamp}" />
```

Comment:
obviously wrong here, should read `newTimestamp: newTimestamp` but I don't see the relation with the PR regarding compatibility with PHP 8.

Reply:
You are right, this is not a php 8 issue, but a TYPO3 11 one.
I had to redo the mark a file as new, because TYPO3 11 does not allow to set "free" file properies anymore. like is was before this PR.
See my changes in this commit:
https://github.com/xperseguers/t3ext-file_list/commit/1d02bc18e13159d1ca0e794837c06fe7385f1b46